pydci/pydci.py

```diff
@@ -64,19 +64,31 @@ def query(self, q, k, max_retrieve, max_composite_visit):
             d-dimensional query points
         k : int
             Number of neighbors to return
-        max_retrieve : int
-            The maximum number of candidates to pull from each composite index
-        max_composite_visit : int
-            The maximum number of times to visit each composite index
+        max_retrieve_const : float
+            Scaling factor for maximum number of candidates to pull from each
+            composite index. Values in the range of 1-10 work well.
```

Comment:
Is this parameter range approximately correct, @cahaselby?

Reply:
yes